config/_default/menu.fa.toml

```diff
@@ -0,0 +1,32 @@
+[[main]]
+name = "انجمن"
+weight = 20
+url = "https://community.letsencrypt.org/"
+[[main]]
+name = "حمایت مالی"
+weight = 30
+identifier = "donate"
+url = "/fa/donate/"
+[[main]]
+name = "درباره ما"
+weight = 40
+identifier = "about"
+url = "/fa/about/"
+[[main]]
+name = "گروه تحقیقاتی امنیت اینترنت (ISRG)"
```

Comment:
As it's the name of an organization, it's better to not translate it if possible. If the translation is really needed, could you explain why so we can make the correct decision?

```suggestion
name = "Internet Security Research Group (ISRG)"
```